ci/launch_cluster.sh

```diff
@@ -38,9 +38,12 @@ template_parameters:
     AdminLocation: 0.0.0.0/0
     PublicSlaveInstanceCount: 1
     SlaveInstanceCount: 5
+    LicenseKey: $DCOS_LICENSE
 EOF
 
-if ! ./dcos-launch create; then
+./dcos-launch create
```

Comment:
I did this because dcos-launch was failing for me with validation error but the output of dcos-launch was not in the output of jenkins run

I saw only this:
./dcos-launch create -L debug
+ exit 2

Reply:
strange...